chapter04/qli/ex1.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+#use List::Util 'sum';
+
+sub total {
+#  my $total = sum(@_);
```

Comment:
Yes, I commented out as I thought it's a bit cheating, the task may want us to implement our own "sum" method. haha